static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -81,7 +82,44 @@ const epicEl = () => {
     return container;
 }
 
-const buildEpicPayload = () => {
+const hasOptedOutOfArticleCount = () =>
+    !!getCookie(ARTICLES_VIEWED_OPT_OUT_COOKIE.name)
+
+const DAILY_ARTILCE_COUNT_KEY = 'gu.history.dailyArticleCount';
```

Comment:
Change ARTILCE &rarr; ARTICLE

```suggestion
const DAILY_ARTICLE_COUNT_KEY = 'gu.history.dailyArticleCount';
```

Reply:
good spot! 👍